libs/shared-ui/src/lib/buy-course-card/buy-course-card.tsx

```diff
@@ -0,0 +1,99 @@
+import Button, { ColorsButton } from '../button/button';
+import Icon from '../icon/icon';
+import styles from './buy-course-card.module.scss';
+
+/* eslint-disable-next-line */
+export interface BuyCourseCardProps {
+  price: number;
+  discount: number;
+  achievements: number;
+  lectures: number;
+  enrolled: number;
+  language: string;
+  skillLevel: string;
+  certificate: boolean;
+}
+
+export function BuyCourseCard(props: BuyCourseCardProps) {
+  const {
+    price,
+    discount,
+    achievements,
+    lectures,
+    enrolled,
+    language,
+    skillLevel,
+    certificate
+  } = props;
+
+  return (
+    <div className={styles['container']}>
+      <div className={`${styles['buy-card']} card`}>
+        <img
+          src="https://magazine.startus.cc/wp-content/uploads/2018/07/bitcoin-2643159_1920-e1533112613226.jpg"
+          alt=""
+        />
+        <div className={`${styles['summary']}`}>
+          <div className={styles['price-promo']}>
+            <div className={styles['price']}>
+              <h3>${price/100}</h3>
+            </div>
+            <div className={styles['discount']}>
+              <span>{discount}% off</span>
+            </div>
+          </div>
+          <div className={styles['actions']}>
+            <Button title="BUY NOW" color={ColorsButton.primary} full={true} />
```

Comment:
the component should has a prop: "onClickBuy" function to catch the event in the parent component